src/lambda-responses/json/index.js

```diff
@@ -9,6 +9,8 @@ export default (statusCode, body, opts) => {
   if (body) {
     if (typeof body === 'object') {
       response.body = JSON.stringify(body);
+    } else if (typeof body === 'string') {
```

Comment:
I think an easier approach to this would be to drop the condition entirely, so we always just do `response.body = JSON.stringify(body)`. That handles strings (in the same way you are explicitly handling them) as well as all of the other test cases (e.g. boolean/number currently drop into the `else` branch unnecessarily).

Reply:
Hmm yeah... however, I think "123" as a string is correct for a number encoded as JSON that will be decoded to a number (i.e. `JSON.parse('123') === 123`). Same applies for boolean values (`JSON.parse('true') === true`)

Reply:
I'm not aware of the constraints/conditions that lead to this implementation  - it does seem a pragmatic approach. Having said that...

If we take a step back this reponse handler implicitly deals with sending JSON data. This would suggest that it should idealy not be interogating the data to deterime whether it is or isn't.

To enable this one would push that determination down the stack. One would initially determin a structured JSON response e.g All our endpoints will responed `{ data: ... }`. The layer below this response handler would have more contextual information and would implement `toJson()` style helper (or alternatives). It knows the client want's JSON it knows its own  internal data structure and whay would be required to return as such. 

So if the layer below always has stringify-able object it would return `{ data: sourceOkbject  }', if its a string `{ data: "" }`, if its a number `{ data: 999 }.

Given that the handler would be reduced to just `response.body = JSON..stringify(body) // { data:.... }` and if it were to encounter an parsing error then it should  throw as that is not what client asked for and the lower parts of the stack didn't pay attention.

Reply:
> Given that the handler would be reduced to just response.body = JSON..stringify(body)

That is what it has been reduced to ;)